SU2_CFD/include/solvers/CScalarSolver.inl

```diff
@@ -149,6 +163,10 @@ void CScalarSolver<VariableType>::Upwind_Residual(CGeometry* geometry, CSolver**
   else
     AD::StartNoSharedReading();
 
+  bool advection{false};
+  
+  if(config->GetKind_Upwind_Species() == UPWIND::SCALAR_ADVECTION) advection = true;
```

Comment:
I think a better name would be BOUNDED_SCALAR, see
https://www.openfoam.com/documentation/guides/latest/doc/guide-schemes-divergence.html